app/design/adminhtml/default/default/template/oyst/oneclick/logactions.phtml

```diff
@@ -0,0 +1,31 @@
+<div>
```

Comment:
Add usual [file header](https://github.com/oystparis/oyst-1click-magento/blob/master/app/design/frontend/base/default/template/oyst/oneclick/oneclick.phtml#L1).

Reply:
@rus-daniel you forget this?